druid/src/widget/clip_box.rs

```diff
@@ -335,72 +354,80 @@ impl<T, W: Widget<T>> ClipBox<T, W> {
             constrain_vertical: false,
             must_fill: false,
             old_bc: BoxConstraints::tight(Size::ZERO),
+            old_size: Size::ZERO,
             managed: false,
         }
     }
 
-    /// Changes the viewport offset by `delta`.
+    /// Pans by `delta` units.
     ///
-    /// Returns true if the offset actually changed. Even if `delta` is non-zero, the offset might
-    /// not change. For example, if you try to move the viewport down but it is already at the
-    /// bottom of the child widget, then the offset will not change and this function will return
-    /// false.
-    pub fn pan_by(&mut self, delta: Vec2) -> bool {
-        self.pan_to(self.viewport_origin() + delta)
+    /// Returns `true` if the scroll offset has changed.
+    pub fn pan_by<'a, C: ChangeCtx<'a>>(
+        &mut self,
+        ctx: &mut C,
+        data: &T,
+        env: &Env,
+        delta: Vec2,
+    ) -> bool {
+        self.with_port(ctx, data, env, |_, port| {
+            port.pan_by(delta);
+        })
     }
 
-    /// Changes the viewport offset on the specified axis to 'position'.
+    /// Pans the minimal distance to show the target rect.
```

Comment:
Is this *rect* referencing the `region` variable? If so, lets change it to `region`.